### PR TITLE
Separates initramfs and rootfs

### PR DESCRIPTION
**Pull request by WiktorOgrodnik**
-adds two phases of compilation: first initramfs + kernel + bootloader,
 second: rootfs + tools (python, git, pip, ...)
-speeds up Linux boot time to a few seconds
-more flexible approach: adding new tools to rootfs will not
 significantly increase boot time

### CODE REVIEW

File: action/run-in-renode.py [hunk 1]
```diff
@@ -511,20 +529,6 @@ def setup_renode():
...
-
-        # increase git buffers
-        # mitigates issues with broken pipe `Send failure: Broken pipe`
-        run_cmd(child, "#", 'git config --global http.maxRequestBuffer 240M')
-        run_cmd(child, "#", 'git config --global http.postBuffer 100M')
-        run_cmd(child, "#", 'git config --global core.compression 0')
-
-        run_cmd(child, "#", 'git config --global --unset https.proxy')
-        run_cmd(child, "#", 'git config --global --unset http.proxy')
-        run_cmd(child, "#", 'git config --global ssl.Verify false')
```
Comment (kbieganski) [inline action/run-in-renode.py:526]: Why was this removed?

File: br2-external/rootfs/board/sifive/hifive-unleashed/post-image.sh
Comment (kbieganski) [inline br2-external/rootfs/board/sifive/hifive-unleashed/post-image.sh:1]: Could use some comments on what it does, and why it does it. I know it's not complicated, but right now you have to take the time to understand it.

File: scripts/compile.sh [hunk 2]
```diff
@@ -7,10 +7,23 @@ make BR2_EXTERNAL=../br2-external hifive_unleashed_defconfig && \
...
+rm -rf output/target && \
+find output/ -name ".stamp_target_installed" -delete && \
+rm -f output/build/host-gcc-final-*/.stamp_host_installed && \
+make BR2_EXTERNAL=../br2-external/rootfs hifive_unleashed_defconfig && \
+make -j$(nproc)
+
+cd ..
+
+cp buildroot/output/images/rootfs.img images
+rm -rf buildroot
```
Comment (kbieganski) [inline scripts/compile.sh:27]: Again, some comments would be nice. Especially on why it's needed.

**Review by kbieganski — CHANGES_REQUESTED** [review]

File: action/run-in-renode.py (hunk 1 above)
Comment (WiktorOgrodnik) [reply action/run-in-renode.py:526]: I think this is not required. I added this because I think it will fixed the issues with https connections, but I do not think that this mitigations even does something.

**Review by kbieganski — APPROVED** [review]
